tests/pact/gas.repl

```diff
@@ -3,7 +3,7 @@
 ; ============================================================ ;
```

Comment:
Where is V1 coverage?

Reply:
Good point! I can create a small test here for this.